multiversx_sdk_cli/tests/test_cli_deps.py

```diff
@@ -0,0 +1,48 @@
+import shutil
+from pathlib import Path
+
+import pytest
+
+from multiversx_sdk_cli.cli import main
+
+
+def test_deps_install_rust():
+    return_code = main(["deps", "install", "rust", "--overwrite"])
+    assert True if return_code == 0 else False
+
+
+def test_deps_check_rust():
+    return_code = main(["deps", "check", "rust"])
+    assert True if return_code == 0 else False
+
+    which_rustc = shutil.which("rustc")
+    if which_rustc:
```

Comment:
I think these `if`s make the test a bit more weak. We can drop the `if`s to make it more restrictive.